odd-platform-api/src/test/java/org/opendatadiscovery/oddplatform/api/ingestion/utils/IngestionModelGenerator.java

```diff
@@ -51,7 +51,9 @@ public static Map<String, Object> generateMetadataMap() {
     }
 
     public static List<DataSetField> generateDatasetFields(final int size) {
-        return EASY_RANDOM.objects(DataSetField.class, size).toList();
+        return EASY_RANDOM.objects(DataSetField.class, size)
+            .peek(dsf -> dsf.setEnumValues(null))
```

Comment:
I'd rather set up an EASY_RANDOM instance so that it wouldn't create `enumValues`